src/canopy-webpack-config.js

```diff
@@ -3,6 +3,12 @@ const CleanWebpackPlugin = require('clean-webpack-plugin')
 const merge = require('webpack-merge')
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin
 
+let isDevServer = false
+
+if (process.argv.some(arg => arg.includes('webpack-dev-server'))) {
```

Comment:
Instead of having people pass in `--env.dev` when they are already running the dev server, i just detect that it's the dev server